modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -14,7 +14,15 @@ Working with {prod-short} in your browser involves multiple URLs:
 * [.underline]#The URLs for starting a new workspace#
 * The URLs of your workspaces in use
 
-With {prod-short}, you can visit a URL in your browser to start a new workspace that contains a clone of a Git repository. This way, you can clone a Git repository that is hosted on GitHub, a GitLab instance, or a Bitbucket server.
+With {prod-short}, you can visit a URL in your browser to start a new workspace that contains a clone of a Git repository. You can use either an HTTPS or SSH URL to start a new workspace with a clone of a Git repository.
+
+[IMPORTANT]
+====
+//TODO: point to the dashboard SSH procedure when https://github.com/eclipse/che/issues/22111 is done.
+* If an SSH URL is used to start a new workspace, the SSH key must be propagated, see link:https://github.com/devfile/devworkspace-operator/blob/main/docs/additional-configuration.adoc#configuring-devworkspaces-to-use-ssh-keys-for-git-operations[Configuring DevWorkspaces to use SSH keys for Git operations].
```

Comment:
This point should be stripped or reworked for downstreaming; I don't think we want to point to the upstream github repo for docs in downstream. 

Maybe we can adopt the `Configuring DevWorkspaces to use SSH keys for Git operations` section into Che docs? The file is already in `.adoc` format, though would need a style pass, etc.

Reply:
I don't think we need to have a dedicated page about configuring SSH by a secret. When https://github.com/eclipse/che/issues/22111 is done we will point to the user friendly dashboard procedure, so the current procedure will be redundant as  too complicated one. @ibuziuk **WDYT?**

Reply:
if I understand it correctly, you'll make this part visible later?

Reply:
got it! thanks for the clarification